### DISCUSSION TITLE
Doesn't it work with the .js file?

### DISCUSSION THREAD

**Discussion by juliam73** [roxiness/routify#523] [Q&A] [answered]
-------- test.js  --------------
import { goto } from '@roxi/routify';

export const goTest = () => {
    $goto('/');
};

------------------------------

It works well with the .svelte file.
but doesn't it work with .js file.

[ routify ver 2.18.8 ]

**Comment by jakobrosenberg** [comment, marked as answer]
`$` has no meaning in vanilla JS, but when you use it in .svelte files, it gets compiled into JS.

In JS you have to use it like a normal store, eg.
```javascript
import { goto } from '@roxi/routify';
import { get } from 'svelte/stores'

const $goto = get(goto)
$goto('/')
```

Testing for R3 should be much improved.

**Reply by juliam73** [reply]
oh~ thank you
